src/nbl/video/IOpenGL_FunctionTable.h

```diff
@@ -24,14 +24,27 @@ namespace nbl {
 			static inline constexpr GLenum CLIENT_STORAGE_BIT				= 0x0200;
 			static inline constexpr GLenum BUFFER_IMMUTABLE_STORAGE			= 0x821F;
 			static inline constexpr GLenum BUFFER_STORAGE_FLAGS				= 0x8220;
-			static inline constexpr GLenum CLIENT_MAPPED_BUFFER_BARRIER_BIT = 0x00004000;
+			static inline constexpr GLbitfield CLIENT_MAPPED_BUFFER_BARRIER_BIT = 0x00004000;
 			static inline constexpr GLenum TEXTURE_MAX_ANISOTROPY			= 0x84FE;
 			static inline constexpr GLenum TEXTURE_LOD_BIAS					= 0x8501;
+			static inline constexpr GLenum DEPTH_CLAMP						= 0x864F;
+			static inline constexpr GLenum FRAMEBUFFER_SRGB					= 0x8DB9; // GL_EXT_sRGB_write_control GLES extension (i think we got to require this)
+			// clip control
+			static inline constexpr GLenum UPPER_LEFT						= 0x8CA2;
+			static inline constexpr GLenum ZERO_TO_ONE						= 0x935F;
+
 			//desktop GL only
 			static inline constexpr GLenum TEXTURE_1D						= 0x0de0;
 			static inline constexpr GLenum TEXTURE_1D_ARRAY					= 0x8c18;
 			static inline constexpr GLenum MIRROR_CLAMP_TO_EDGE				= 0x8743;
 			static inline constexpr GLenum MIRROR_CLAMP_TO_BORDER			= 0x8912;
+			static inline constexpr GLenum DOUBLE							= 0x140A;
+			static inline constexpr GLenum COLOR_LOGIC_OP					= 0x0BF2;
+			static inline constexpr GLenum PRIMITIVE_RESTART				= 0x8F9D;
+			static inline constexpr GLenum MULTISAMPLE						= 0x809D;
+			static inline constexpr GLenum POLYGON_OFFSET_POINT				= 0x2A01;
+			static inline constexpr GLenum POLYGON_OFFSET_LINE				= 0x2A02;
+			static inline constexpr GLenum TEXTURE_CUBE_MAP_SEAMLESS		= 0x884F;
```

Comment:
wtf?

Reply:
IOpenGL_FunctionTable (base for GL and GLES func table) includes gles3.2 header (cant see symbols from glext.h and gles extensions header here)

Reply:
this sort of redefinition of OpenGL constant enums is just asking for trouble.

Reply:
did it get removed